runcoms/README.md

```diff
@@ -3,68 +3,142 @@ Configuration Files
 
 Zsh has several system-wide and user-local configuration files.
 
-Prezto has one user-local configuration file.
+Prezto has one user-local configuration file: `.zpreztorc`
 
 System-wide configuration files are installation-dependent but are installed
-in */etc* by default.
+in `/etc` by default.
 
 User-local configuration files have the same name as their global counterparts
 but are prefixed with a dot (hidden). Zsh looks for these files in the path
-stored in the `$ZDOTDIR` environmental variable. However, if said variable is
-not defined, Zsh will use the user's home directory.
+stored in the `$ZDOTDIR` environment variable. However, if said variable is
+not defined, Zsh will use the user's home directory. This is found on the
+config scripts as `${ZDOTDIR:-$HOME}` and is a `zsh` idiomatic way of doing
+something like this:
 
-File Descriptions
------------------
-
-The configuration files are read in the following order:
-
-  01. /etc/zshenv
-  02. ~/.zshenv
-  03. /etc/zprofile
-  04. ~/.zprofile
-  05. /etc/zshrc
-  06. ~/.zshrc
-  07. ~/.zpreztorc
-  08. /etc/zlogin
-  09. ~/.zlogin
-  10. ~/.zlogout
-  11. /etc/zlogout
-
-### zshenv
-
-This file is sourced by all instances of Zsh, and thus, it should be kept as
-small as possible and should only define environment variables.
-
-### zprofile
 
-This file is similar to zlogin, but it is sourced before zshrc. It was added
-for [KornShell][1] fans. See the description of zlogin below for what it may
-contain.
+```bash
+if [[ -n $ZDOTDIR ]]; then
+   DOTFILES_PATH=$ZDOTDIR
```

Comment:
Minor nit: can you remove a space from the beginning here to be consistent with the `else` branch?